runtime/manta-pc/src/lib.rs

```diff
@@ -212,6 +212,27 @@ impl pallet_transaction_payment::Config for Runtime {
 	type FeeMultiplierUpdate = SlowAdjustingFeeUpdate<Self>;
 }
 
+parameter_types! {
+	// The maximum weight that may be scheduled per block for any
+	// dispatchables of less priority than schedule::HARD_DEADLINE.
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) *
+		RuntimeBlockWeights::get().max_block;
+	// The maximum number of scheduled calls in the queue for a single block.
+	// Not strictly enforced, but used for weight estimation.
+	pub const MaxScheduledPerBlock: u32 = 50;
```

Comment:
why 50?

Reply:
The parameters here are a reference to the configuration parameters of `pallet_schedule` in `substrate`. Theoretically, it is possible to modify